### ISSUE TITLE
Add sentinel tables to facilitate speed measurement

### ISSUE THREAD

**Issue by datacharmer** [datacharmer/test_db#1]
Adding a "sentinel" table that will allow for a quick calculation of performance speed, and also a quick evaluation of replication efficiency.
There should be an empty  "first_table" before any other operation on the schema, and a "last_table" after all the data has been loaded. 
The method is clearly explained in the article [Quick benchmarking trick](http://datacharmer.blogspot.com.es/2011/03/quick-benchmarking-trick.html)

**Comment by datacharmer** [comment]
Rather than adding first/last tables to calculate the time
elapsed for loading, we use the creation time of the first table (employees)
and the update time of the last table (salaries) to calculate the time.

A similar operation was added to the test_* files, so that they can
show how long the checksum operation takes.

**Comment by Delpedro** [comment]
Hi,

Work Environment
Windows 8.1 VM in Virtualbox

I have downloaded these files as a zip, unzipped them and have them located in the downloads folder.

Method 1
I have MySQL installed c/w Workbench - I cannot import this sample DB via the Workbench, it will not work. The tables etc import but are empty.

Method 2
I'm trying to install the db via the MySQL Cli using the source command i.e. - I have tried all these commands & getting the errors as per below screen grab. I have seen many users having these problems, with the suggested solution being, missing files, download the dump files. However, I have downloaded your files, and they work fine in a Docker container within a Ubuntu VM......

source C:Users\Downloads\employees.sql
source C:Users\Downloads\employees_partitioned.sql
source C:Users\Downloads\employees_partiitioned_5.1.sql

![mysql_employee_error](https://cloud.githubusercontent.com/assets/6932157/14569344/1856ecb0-0336-11e6-8f76-fba2a02ad3eb.png)

Any help would be very much appreciated. 

Thank you.

Del.
